SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -185,16 +189,16 @@ void ActorSystem::ProcessAdds(const FEntitySubViewUpdate& SubViewUpdate)
 				continue;
 			}
 
-			if (!PresentEntities.Contains(Delta.EntityId))
+			if (!PresentEntities.Contains(EntityId))
 			{
 				// Create new actor for the entity.
-				EntityAdded(Delta.EntityId);
+				EntityAdded(EntityId);
 
-				PresentEntities.Emplace(Delta.EntityId);
+				PresentEntities.Emplace(EntityId);
 			}
 			else
 			{
-				RefreshEntity(Delta.EntityId);
```

Comment:
hm, do we / does native call postnetreceive on an actor being created?

Reply:
Yep, the PostNetReceives are being called in ApplyFullState which is called from both EntityAdded and  RefreshEntity as well